.github/workflows/deply.yml

```diff
@@ -0,0 +1,55 @@
+name: Deploy Jekyll site to Pages
+
+on:
+  push:
+    branches: ["main"]
```

Comment:
can we make this a "production" branch?

Reply:
Minor preference for "prod", since that is what I'm used to. Then again I also prefer mainline over main, so it might just be my preference for "Amazon" stuff.

Reply:
Switched to `prod`.